CMakeLists.txt

```
@@ -29,6 +30,9 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
 include(TargetArch)
 detect_target_architecture(_TARGET_ARCH)
 
+CHECK_INCLUDE_FILE("<getopt.h>" HAVE_GETOPT_H)
```

```suggestion
CHECK_INCLUDE_FILE(getopt.h HAVE_GETOPT_H)
```

I misremembered the check include syntax, sorry 😅

---

It's just esthetics, but you can lower case the function. (CMake is case insensitive for functions/macros, but case sensitive for variables)

---

yeah, I know. However, I've been lazy and copy pasted the function name…